go.mod

```diff
@@ -6,8 +6,11 @@ require (
 	github.com/FactomProject/btcutilecc v0.0.0-20130527213604-d3a63a5752ec // indirect
 	github.com/Kubuxu/gocovmerge v0.0.0-20161216165753-7ecaa51963cd
 	github.com/StackExchange/wmi v0.0.0-20181212234831-e0a55b97c705 // indirect
-	github.com/TRON-US/go-btfs-cmds v0.1.3
+	github.com/TRON-US/go-btfs-cmds v0.1.4
 	github.com/TRON-US/go-btfs-config v0.1.6
+	github.com/TRON-US/go-mfs v0.2.0
```

Comment:
can you run `go mod tidy`? I think `github.com/ipfs/go-mfs` should have been removed? Or is there another package using ipfs/go-mfs as indirect?

Reply:
I ran. github.com/TRON-US/go-btfs/core is using go-mfs according to "go mod why".

Reply:
It should not use `ipfs/go-mfs` anymore. Can you find out which file is still using it?

Reply:
No one from go-btfs. 
In go.mod, we have.
github.com/ipfs/go-mfs v0.1.0 // indirect